fs/ext4/inode.c

```diff
@@ -3835,7 +3835,7 @@ static int __ext4_block_zero_page_range(handle_t *handle,
 		/* Uhhuh. Read error. Complain and punt. */
 		if (!buffer_uptodate(bh))
 			goto unlock;
-		if (decrypt) &&
+		if ((decrypt) &&
```

Comment:
I think you should just delete the parenthesis following `decrypt` than adding this one.